docs/source/user_guide/model_understanding.ipynb

```diff
@@ -91,6 +91,35 @@
     "pipeline_binary.graph_feature_importance()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we have a linear model, we can also view feature importance by simply inspecting the coefficients of the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from evalml.model_understanding import get_linear_coefficients\n",
+    "\n",
+    "pipeline_linear = BinaryClassificationPipeline(component_graph = {\n",
+    "            \"Label Encoder\": [\"Label Encoder\", \"X\", \"y\"],\n",
+    "            \"Imputer\": [\"Imputer\", \"X\", \"Label Encoder.y\"],\n",
+    "            \"Logistic Regression Classifier\": [\n",
+    "                \"Logistic Regression Classifier\",\n",
+    "                \"Imputer.x\",\n",
+    "                \"Label Encoder.y\",\n",
+    "            ],\n",
+    "        })\n",
+    "pipeline_linear.fit(X_train, y_train)\n",
+    "\n",
+    "get_linear_coefficients(pipeline_linear.estimator, features=X.columns)"
```

Comment:
nit: Not sure if this would be good as a change to the function output or just a small change to the cell, but I think it'd look cleaner and make more sense for OS users if we showed this as a dataframe with column names labeled:
<img width="706" alt="image" src="https://user-images.githubusercontent.com/22552445/162016845-b3aca197-9c71-4faa-a3b6-ceac6ef5e592.png">

Similar to 
<img width="365" alt="image" src="https://user-images.githubusercontent.com/22552445/162016882-ebec3d7c-e629-4835-b66c-433ca7d5e2db.png">